administrator-manual/en/docker.rst

```diff
@@ -70,11 +105,15 @@ For example (pihole on aqua with a static IP): ::
 Aeria network
 =============
 
+.. note::
+
+  This network is not standard on docker, the developer can be contacted at https://github.com/devplayer0/docker-net-dhcp
+
```

Comment:
Please move this phrase
>  the developer can be contacted at https://github.com/devplayer0/docker-net-dhcp

To a link at the end of the page, not inside an highlighted note.

Reply:
done